add-database

Kazantsevyury opened the pull request:
Implement database interaction using H2

Kazantsevyury committed:
feat: Implement database interaction using H2

Kazantsevyury committed:
feat: Implement database interaction using H2

Kazantsevyury committed:
feat: Implement database interaction using H2

Kazantsevyury committed:
feat: Implement database interaction using H2

Kazantsevyury committed:
feat: Implement database interaction using H2

Kazantsevyury committed:
feat: Implement database interaction using H2

PolnySkvorets commented on src/main/java/ru/yandex/practicum/filmorate/controller/FilmController.java:
```diff
@@ -2,87 +2,111 @@
+    @PostMapping("/films")
+    public Film createFilm(@Valid @RequestBody Film film) {
+        log.info("Creating a new film");
+        log.info(film.toString());
```
Лучше эти логи в один текст объединить

PolnySkvorets replied:
Посмотри, где такое повторяется

PolnySkvorets commented on src/main/resources/data.sql:
```diff
@@ -0,0 +1,32 @@
+insert into rating(rating_name)
+values ('G');
```
Ты можешь добавить через запятую сразу все значения

PolnySkvorets reviewed (commented):
Привет! Оставил пару замечаний, в остальном хорошо

Kazantsevyury committed:
refactor: Optimize SQL inserts for movie ratings and genres

Consolidate multiple INSERT INTO statements into single statements for better efficiency and readability.

Additionally, optimize logging in FilmController class.

Kazantsevyury committed:
refactor: Optimize SQL inserts for movie ratings and genres

Consolidate multiple INSERT INTO statements into single statements for better efficiency and readability.

Additionally, optimize logging in FilmController class.

Kazantsevyury committed:
refactor: Optimize SQL inserts for movie ratings and genres

Consolidate multiple INSERT INTO statements into single statements for better efficiency and readability.

Additionally, optimize logging in FilmController class.

PolnySkvorets commented on src/main/resources/data.sql:
```diff
@@ -1,32 +1,14 @@
-insert into genre(genre_name)
-values ('Боевик');
\ No newline at end of file
+INSERT INTO rating (rating_name) VALUES
```
Проверил это замечаний, ошибся, лучше вернуть как было

PolnySkvorets reviewed (commented):
Привет! Оставил один коммент, нужно откатить одну правку

Kazantsevyury committed:
refactor: update data.sql

PolnySkvorets reviewed (approved):
Принято. Отлично
